src/main/java/seedu/address/logic/autocomplete/AutoCompleteNusNetId.java

```diff
@@ -0,0 +1,53 @@
+package seedu.address.logic.autocomplete;
+
+import java.util.Arrays;
+import java.util.regex.Matcher;
+import java.util.regex.Pattern;
+
+import seedu.address.commons.util.Trie;
+
+/**
+ * AutoComplete for NUSNET ID
+ */
+public class AutoCompleteNusNetId implements AutoComplete {
+    private static Trie nusNetIdTrie;
+    private static final Pattern NUSNET_ID_FORMAT = Pattern.compile("nn/(.*)");
```

Comment:
Is it possible to use the constant from `CliSyntax`?
```suggestion
    private static final Pattern NUSNET_ID_FORMAT = Pattern.compile(PREFIX_NUSNET.getPrefix() + "(.*)");
```

Reply:
Will do